assets/js/hooks/useInViewSelect.js

```diff
@@ -27,6 +27,10 @@ import { useCallback, useRef } from '@wordpress/element';
  */
 import { useInView } from './useInView';
 
+const notInViewCallback = () => {
+	return undefined;
+};
```

Comment:
Are there specific reasons why it must be an arrow function? We should use the standard function declaration whenever possible.

```suggestion
function notInViewCallback() {
	return undefined;
}
```

Reply:
Got it, changed.

Reply:
Got it, changed.